### ISSUE TITLE
New release?   Jupyter-lab broken with pinned Jinja2

### ISSUE THREAD

**Issue by matthew-brett** [open-dicom/dicom_parser#87]
Would you consider making a new release?

I just pip-installed Dicom_parser, and JupyterLab, and this gave me a broken JupyterLab:

```bash
$ mkvirtualenv test
$ pip install jupyterlab
$ pip install dicom_parser
```

This ends with output:

```
Installing collected packages: numpy, pydicom, pandas, nibabel, jinja2, dicom-parser
  Attempting uninstall: jinja2
    Found existing installation: Jinja2 3.0.3
    Uninstalling Jinja2-3.0.3:
      Successfully uninstalled Jinja2-3.0.3
Successfully installed dicom-parser-1.1.1 jinja2-2.11.3 nibabel-3.2.2 numpy-1.22.2 pandas-1.4.1 pydicom-2.1.2
```

Notice the downgrade of Jinja.   Next:

```
$ jupyter-lab
```

Gives:

```
[snip]
ImportError: cannot import name 'soft_unicode' from 'markupsafe' (/Volumes/zorg/mb312/.virtualenvs/test/lib/python3.8/site-packages/markupsafe/__init__.py)
```

Web search gives [this thread](https://github.com/pallets/markupsafe/issues/282), which suggests:

```
pip install -U jinja2
```

Sure enough, this fixes it.    

Repeating these steps, but installing from latest Master locally, does not downgrade Jinja2, and does not cause this error. 

Maybe a new release would prevent this error?

**Comment by ZviBaratz** [comment]
Certainly. Do you think I should wait until we merge the existing PR or just go for it?

**Comment by matthew-brett** [comment]
Maybe just release as is, so the PR has time to settle after it gets
merged.

**Comment by ZviBaratz** [comment]
Published [v1.2.0](https://github.com/open-dicom/dicom_parser/releases/tag/v1.2.0).
